shoop/front/basket/update_methods.py

```diff
@@ -14,6 +14,12 @@
 from shoop.utils.numbers import parse_decimal_string
 
 
+def get_error_text(error):
+    if hasattr(error, "__iter__"):
```

Comment:
Okay. I mean, the thing is even `strings` have `__iter__`, so if you passed in a string, you'd get `s,o,m,e,t,h,i,n,g,b,a,d` out. :D